pkg/configure/cmd.go

```diff
@@ -1,35 +1,65 @@
 package configure
 
 import (
+	"path/filepath"
+
 	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/attributes/container"
 	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/attributes/pod"
-	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/config_files"
-	"github.com/sirupsen/logrus"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/enrichment/metadata"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/ca"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/conf"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/curl"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/pmc"
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/preload"
+	"github.com/go-logr/logr"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 )
 
 const (
-	configDirFlag = "config-directory"
+	inputFolderFlag  = "input-directory"
+	configFolderFlag = "config-directory"
+	installPathFlap  = "install-path"
 )
 
 var (
-	configDirectory string
+	inputFolder  string
+	configFolder string
+	installPath  = "/opt/dynatrace/oneagent"
 )
 
 func AddFlags(cmd *cobra.Command) {
-	cmd.PersistentFlags().StringVar(&configDirectory, configDirFlag, "", "(Optional) Path where enrichment/configuration files will be written.")
+	cmd.PersistentFlags().StringVar(&inputFolder, inputFolderFlag, "", "(Optional) Base path where to look for the configuration files.")
+
+	cmd.PersistentFlags().StringVar(&configFolder, configFolderFlag, "", "(Optional) Base path where to put the configuration files.")
+
+	cmd.PersistentFlags().StringVar(&installPath, installPathFlap, "/opt/dynatrace/oneagent", "(Optional) Base path where the agent binary will be put.")
 
 	container.AddFlags(cmd)
 	pod.AddFlags(cmd)
-
 }
 
-func Execute(fs afero.Afero) error {
-	if configDirectory == "" {
+func Execute(log logr.Logger, fs afero.Afero, targetDir string) error {
+	if configFolder == "" || inputFolder == "" {
```

Comment:
Should they really be coupled like this? I guess this is because `inputfolder` is needed for the config files you added isn't it?